R/utils.R

```diff
@@ -163,3 +163,21 @@ set_element_if_null <- function(x, element, value) {
 
   x
 }
+
+# fails ------------------------------------------------------------------------
+#' Does the Evaluation of an Expression Fail?
+#'
+#' @param expr expression to be evaluated within \code{try(\dots)}
+#' @param silent passed to \code{\link{try}}, see there.
+#' @return \code{TRUE} if evaluating \code{expr} failed and \code{FALSE} if
+#'   the evalutation of \code{expr} succeeded.
+fails <- function(expr, silent = TRUE) {
```

Comment:
Could you please add the `#' @noRd` tag?

Reply:
@maelle: Should be fixed now...

Reply:
thanks!